### PR TITLE
E3Conv as a message passing graph conv in pytorch geometric

### PR DESCRIPTION
**Pull request by blondegeek**
Here's my first attempt at an implementation of our convolutions as a message passing network in pytorch geometric. I'm not sure if timing wise it will be fast enough, but it seems to be a lot easier than dealing with rectangular neighbor lists.

### CODE REVIEW

File: e3nn/message_passing.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import torch
+from torch_geometric.nn import MessagePassing
+
+
+class E3Conv(MessagePassing):
+    def __init__(self, Kernel, Rs_in, Rs_out):
+        super(E3Conv, self).__init__(aggr='add')
```
Comment (mariogeiger) [inline e3nn/message_passing.py:7]:
```suggestion
        super().__init__(aggr='add')
```
for uniformity with the rest of the code

File: e3nn/message_passing.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import torch
+from torch_geometric.nn import MessagePassing
```
Comment (mariogeiger) [inline e3nn/message_passing.py:2]: add `torch-geometric` as dependency? @bkmi

Comment (bkmi) [reply e3nn/message_passing.py:2]: I suppose we could have it as an extra?

It seems like a lot to add for this one single use.

**Comment by mariogeiger** [comment]
We will soon use pytorch-sparse as well

**Comment by bkmi** [comment]
Are you talking about
https://github.com/rusty1s/pytorch_sparse
or 
https://pytorch.org/docs/stable/sparse.html?

**Comment by mariogeiger** [comment]
rusty1s/pytorch_sparse

I think the other one is not working

**Pull request closed without merging**